docs/starsolo.rst

```diff
@@ -6,7 +6,7 @@ This ``starsolo_workflow`` workflow generates gene-count matrices from FASTQ dat
 Prepare input data and import workflow
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. Run ``cellranger_workflow`` to generate FASTQ data
+1. Run ``starsolo_workflow`` to generate FASTQ data
```

Comment:
`cellranger_workflow` must be used for this `mkfastq` step, as STARsolo starts from FASTQ files.